tests/Database/FilesystemTests/DBTests/TransactionsDBTests/SpendableTest.nim

```diff
@@ -0,0 +1,170 @@
+#TransactionsDB Spendable Test.
```

Comment:
Appreciate removing the redundant Tests suffix from the folders. Why was it left on for FS*, DB*, and TDB*?

Reply:
Simply overlooked. Will fix.